website/docs/Pipelines/reblock_gvcf_spec.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 14
+---
+
+# Reblocked GVCF Technical Overview
+The following is a technical overview of the current reblocked GVCF schema (gnomADv4) as well descriptions of the previous GVCF schemas.
+
+## gnomADv4 schema (4 bands)
+* New “current” as of August 2021
+* Arguments -do-qual-approx --floor-blocks -GQB 20 -GQB 30 -GQB 40 
+
+#### Summary (improvements compared with HC GVCFs):
+* NOTE: this format is currently incompatible with GenotypeGVCFs!  Only GnarlyGenotyper can be specified in Joint genotyping WDL input json.
+* PLs are omitted for homozygous reference sites to save space (which is why this format is currently incompatible with GenotypeGVCFs) -- GQs are output for genotypes, PLs can be approximated as [0, GQ, 2*GQ].
+
+* Compared with old HaplotypeCaller GVCFs, GQ resolution for homozygous reference genotypes is reduced -- i.e. homRef GQs will be underconfident, which may affect analyses like de novo calling where confident reference genotypes are important.
+
+* Alleles that aren’t called in the sample genotype are dropped, i.e. each variant should have no more than two alt alleles (in very rare cases, usually just one plus <NON_REF>)
+New annotations to enable merging data for filtering without using genotypes -- RAW_GT_COUNT(S?) for doing ExcessHet calculation from a sites-only file; QUALapprox and/or AS_QUALapprox for doing QUAL approximation/filling QUAL VCF field from a combined sites-only field; VarDP and/or AS_VarDP used to calculate QualByDepth/QD annotation for VQSR
+* No more MIN_DP -- joint DP varies?
+* InbreedingCoeff values will vary due to some low quality genotypes being rounded down to GQ0
+
+#### Cost/scale improvements
+
+* Storage footprint is reduced compared with HaplotypeCaller output
+* Fewer VariantContexts, i.e. lines, per VCF speeds up GenomicsDB/Hail import
+* Fewer alternate alleles reduces memory requirements for merging
+
+
+### Specific improvements compared with 7-band schema
+* Not dropping GQ0s -- reblocked GVCF should cover all positions (that input GVCF covers)
+* No overlaps -- only overlapping positions should be two variants, i.e. deletions, on separate haplotypes
+(No more no-calls in GVCFs-- all genotypes should be called, positions with no data will be hom ref with GQ0; NOTE that this is not being validated and GenotypeGVCFs outputs will have no-calls to preserve previous behavior)
+* Latest Terra WDL and config at https://app.terra.bio/#workspaces/ccdg-gatk-supp-terra/CCDG_WES_Reblocking_Work_F3/workflows/ccdg-gatk-supp-terra/ReblockGVCF-gatk4_exomes_goodCompression_fixed (NOTE that this has an off-by-one issue for contigs starts of exactly 1, which we don’t see in Broad production)
```

Comment:
I know the public workspace isn't out yet but maybe we don't want to link to a workspace until it is? I think I'd just remove this entire bullet point, this workspace is outdated.